packages/mux-player/src/media-theme-mux.html

```diff
@@ -67,6 +67,19 @@
       --captions-selectmenu: none;
     }
 
+
+    /* 0.433s is the transition duration for VTT Regions.
+     * Borrowed here, so the captions don't move too fast. */
+    media-controller ::slotted([slot='media']) {
+      --media-webkit-text-track-transition: transform 0.433s ease-out 0.3s;
```

Comment:
it feels better but it's still touching the control bar fading out, the issue seems that the media-chrome control bar takes a full second to fade out. that's too long I think.

Reply:
Yeah, we can probably increase the delay a tiny bit more, but probably not a lot more.